yb-voyager/src/tgtdb/yugabytedb_test.go

```diff
@@ -361,3 +362,133 @@ func TestPGStatStatementsQuery(t *testing.T) {
 		})
 	}
 }
+
+func TestCollectPgStatStatements_BasicSelectQueries(t *testing.T) {
+	// Test data: queries and their expected execution counts across nodes
+	testQueries := map[string]struct {
+		text          string
+		parameterized string
+		execCounts    []int // executions per node [node0, node1, node2]
+	}{
```

Comment:
Probably better to create a separate type definition for the test query struct since its being used at multiple places?